libgc/pthread_stop_world.c

```diff
@@ -2,7 +2,7 @@
 
 #if defined(GC_PTHREADS) && !defined(GC_SOLARIS_THREADS) \
      && !defined(GC_IRIX_THREADS) && !defined(GC_WIN32_THREADS) \
-     && !defined(GC_DARWIN_THREADS) && !defined(GC_AIX_THREADS)
+     && !defined(GC_DARWIN_THREADS)
```

Comment:
undo this change in `libgc`, we'll likely have a submodule for boehm GC soon, so this would be lost anyway (/cc @joncham )